mentoring/mentor-guide.md

```diff
@@ -0,0 +1,101 @@
+Mentor Guidelines
+=======
+*WIP*
+
+With over 37 special interest and working groups (SIGs/WGs), the Kubernetes organization is very large to navigate through and no one has all of the answers. Contributors rely on our docs, commenting on PRs/Issues, finding a mentor on their own, and any time they can interact with others to get up to speed. Whether it’s Slack, update meetings, or KubeCons; contributors want to discuss technical specifics, the landscape, or how to be a valuable member. Being a mentor can help accelerate someone's path to being more productive in our ecosystem.
```

Comment:
> the Kubernetes organization is very large to navigate through and no one has all of the answers.
Feels a little weird in the mouth...

thoughts on:
> the Kubernetes ecosystem is vast and no one has all of the answers.
?

Reply:
`KubeCon,` instead of `KubeCon;`

Reply:
ty!

re: becoming README: I'd like the keep the README that is the list of programs we have and have this be just the mentor guide.